contracts/truffle-config.js

```diff
@@ -1,12 +1,11 @@
-const path = require('path');
 const HDWalletProvider = require('truffle-hdwallet-provider');
 const {
   mnemonic,
   mainnetProviderUrl,
   rinkebyProviderUrl,
   goerliProviderUrl,
   kovanProviderUrl,
-} = require('./secretsManager.js');
+} = require('./secretsManagerCi.js');
```

Comment:
Thanks, we basically don't want to accidentally put our mainnet keys into github. Thats why we have the `secretsManager.js` in  the gitignore.

Reply:
pardon me :)

Reply:
No, thanks for the pr, I'll just change this line and accept it